src/shoelace-autoloader.ts

```diff
@@ -35,14 +35,14 @@ export async function discover(root: Element | ShadowRoot) {
  * Registers an element by tag name.
  */
 function register(tagName: string): Promise<void> {
-  const tagWithoutPrefix = tagName.replace(/^sl-/i, '');
-  const path = getBasePath(`components/${tagWithoutPrefix}/${tagWithoutPrefix}.js`);
-
   // If the element is already defined, there's nothing more to do
   if (customElements.get(tagName)) {
     return Promise.resolve();
   }
```

Comment:
Moving this `if` block is a minor optimization... if the element is already registered we don't need to calculate it's tagWithoutPrefix or path